fitbenchmarking/fitbenchmarking
# 103 setup docker

#### Description of Work

Fixes #103 


#### Testing Instructions

1. Check the tests run
2. Check you can run the docker container locally
3.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
